.dockerignore

```diff
@@ -42,6 +42,7 @@ docs/
 install/
 superset-frontend/cypress-base/
 superset-frontend/coverage/
+superset-frontend/.temp_cache/
```

Comment:
this is a big win, that folder (storing the webpack cache) gets real big, like GBs, not bringing it in the image really improve the frontend build stage